### DISCUSSION TITLE
Colors guidelines for automations (MC)

### DISCUSSION THREAD

**Discussion by banand86** [twilio-labs/paste#1430] [General] [answered]
Hey team.
I am designing in the Automations (Marketing Campaigns) space. Looking at redoing the email and logic (x days to go) cards. Part of the redesign includes having color association for logic (time delay) and actions (email) cards for easy scanning. In the mock below you will see use of 2 colors - charting mango (#fdd835) and charting seafoam (#008cd4). These are colors used in carts (as the name suggests). Wondering if there are any guidelines/restriction on using these colors in other parts of the product. 
The colors mentioned above are placeholders and I am happy to change them as per guidelines. I am aware of [SG guidelines page on color](https://styleguide.sendgrid.com/colors.html) but I don't have enough info on it to make a decision.

<img width="822" alt="Screen Shot 2021-05-10 at 10 57 36 AM" src="https://user-images.githubusercontent.com/80920953/117696485-947abf00-b17e-11eb-8824-0a5f680dfb63.png">

Happy to give more context if needed. Thanks team!

**Comment by TheSisb** [comment, marked as answer]
Hi @banand86,

At this time Paste doesn't have any color tokens specific to charting, so we don't have any strong opinions about those specific colors. General guidance to keep in mind: 

1. Make sure color isn't the only differentiator to tell things apart (color blindness)
2. Make sure color is used with an accessible contrast ratio

Out of curiosity, do those colors need to stay consistent over persistent sessions or could they just be random colors each time? This is a helpful way to think about whether you're using color as an associative property to other parts of the website which can be inaccessible to users.

**Reply by banand86** [reply]
Hey Shadi. Thanks for the prompt response.
Looks like I got distracted in the middle of writing back. Just realizing I never hit the reply button! :(

- Those colors will be consistent across automation. Thanks for your input on contrast and yeah I'll make sure color isn't the only differentiator.

**Reply by kate-porter** [reply]
Hey @banand86 - if you get a chance, sync with @kylegilbreath on charting colors. He's working on an insights dashboard right now and is planning to partner with us on getting charting colors into Paste here pretty soon, so he'd be a good person to chat with!
